core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -483,13 +484,13 @@ private SortedMap<KeyExtent,Bulk.Files> computeMappingFromFiles(FileSystem fs, T
     if (this.executor != null) {
       executor = this.executor;
     } else if (numThreads > 0) {
-      executor = service = context.threadPools().getPoolBuilder("client.bulk.load")
+      executor = service = context.threadPools().getPoolBuilder(BULK_IMPORT_CLIENT_LOAD_POOL_NAME)
           .numCoreThreads(numThreads).enableThreadPoolMetrics().build();
     } else {
       String threads = context.getConfiguration().get(ClientProperty.BULK_LOAD_THREADS.getKey());
       executor = service = context.threadPools()
-          .getPoolBuilder(
-              METRICS_BULK_IMPORT_CLIENT_PREFIX + ClientProperty.BULK_LOAD_THREADS.getKey())
```

Comment:
Is it important that we append `bulk.threads` to this thread pool? 
If not, this could also switch to being a constant.

Reply:
In talking with one user, the names of the thread pools did not matter, as long as they could trace the thread pool to its location in the source code to see how its used and what its supposed to be doing.

Reply:
okay, then I'll switch this to a constant.

Reply:
Fixed in 3bcc3f8854